pkgs/data/themes/dracula-theme/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub }:
+
+let
+  themeName = "Dracula";
+  version = "2.0";
+in
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation {
```
Sorry, small change I missed